jenkins/release.groovy

```diff
@@ -0,0 +1,93 @@
+/*
+ * Hibernate Validator, declare and validate application constraints
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+
+@Library('hibernate-jenkins-pipeline-helpers@1.5') _
+
+import org.hibernate.jenkins.pipeline.helpers.version.Version
+
+pipeline {
+	agent {
+		label 'Worker&&Containers'
+	}
+	tools {
+		maven 'Apache Maven 3.8'
+		jdk 'OpenJDK 11 Latest'
```

Comment:
In the particular case of 6.0, I think we should stick to the Oracle JDK 8 as I don't feel like changing the JDK to release this branch.